databags/communityteams_members.json

```diff
@@ -0,0 +1 @@
+{"team_members": [{"name": "Hemanth Kumar", "projects": ["Project Contributor for the License Chooser project, has privileges for the chooser repository"]}, {"name": "Jacqueline Binya", "projects": ["Project Contributor for the License Chooser project, has privileges for the chooser repository"]}, {"name": "Ari Madian", "projects": ["Project Core Committer for the License Chooser project, has privileges for the chooser repository", "Project Collaborator for the CC Search project, has privileges for the cccatalog-frontend repository"]}, {"name": "Mayank Nader", "projects": ["Project Core Committer for the CC Search Browser Extension project, has privileges for the ccsearch-browser-extension repository"]}, {"name": "Mar\u00eda Bel\u00e9n Guaranda Cabezas ", "projects": ["Project Core Committer for the Linked Commons project, has privileges for the cccatalog-dataviz repository"]}, {"name": "Olga Bulat", "projects": ["Project Core Committer for the License Chooser project, has privileges for the chooser repository"]}, {"name": "Ana Paula Rocha", "projects": ["Project Collaborator for the Platform Toolkit project, has privileges for the mp repository"]}, {"name": "Bhumij Gupta", "projects": ["Project Core Committer for the Link Checker project, has privileges for the cc-link-checker repository"]}, {"name": "Ahmad Bilal", "projects": ["Project Core Committer for the WordPress Plugin project, has privileges for the wp-plugin-creativecommons repository"]}, {"name": "Dhruv Bhanushali", "projects": ["Project Core Committer for the Vocabulary project, has privileges for the fonts, vocabulary, vue-vocabulary repositories"]}, {"name": "Chidiebere Onyegbuchulem", "projects": ["Project Member for the Vocabulary project, has privileges for the fonts, vocabulary, vue-vocabulary repositories"]}]}
```

Comment:
You should update the script here to print this more prettily so it's human readable. See https://github.com/creativecommons/ccos-scripts/blob/master/generate_projects_page/generate_projects_page.py#L150